src/util-napatech.c

```diff
@@ -1544,7 +1558,9 @@ uint32_t NapatechSetupTraffic(uint32_t first_stream, uint32_t last_stream)
                 }
                 stream_spec = CONFIG_SPECIFIER_INDIVIDUAL;
 
-                ByteExtractStringUint8(&ports_spec.first[iteration], 10, 0, port->val);
+                if (ByteExtractStringUint8(&ports_spec.first[iteration], 10, 0, port->val) == -1) {
+                    SCLogError("Invalid value in napatech.ports specification in conf file.");
```

Comment:
Looks like error here is just a warning.

Generally, shouldn't the new warnings here be errors that are actually handled as errors? In other words, fail to start up? If they are warnings like this, what happens if the warnings get raised? Does start up still succeed with some built-in defaults?

Reply:
I've changed the SCLogError and SCLogWarning calls to FatalError because there is no sane default value if the port parsing is wrong.